### ISSUE TITLE
Show latest status logs when monitoring

### ISSUE THREAD

**Issue by simlarsen** [OneUptime/oneuptime#725]
I expected to be able to see somewhere the periodic checks - even if they were just green/Operational. So for example in the Status Timeline, I was expecting to see either one row for each check (even if no change), or to be able to see in the logs each individual check. Because this particular monitor hasn't gone down or suffered any degradation I can't see anything at all after initially setting it up - hence my confusion. The point is that there's nowhere on the interface that says (for example) "Last checked at X.XX pm BST", or shows me a log of last check or a status etc. Or at least, I couldn't see this anywhere. I think I've since understood that all I'll see are when things change, but that makes it hard to be confident it's actually monitoring.

In short: I can't see any way to easily visually differentiate between: "It's running but it's just all Operational since initially setting it up" and "You set it up but there's a bug/misconfig somewhere and it's not running at all". No "continuous progress" indicators. Or if there are I can't see them.
